demos/breakout_ai/main.py

```diff
@@ -19,12 +19,20 @@
 class BreakoutAi:
     """This is the breakout_ai demo. It plays the classic Atari game breakout"""
 
-    demo_time = 120
```

Comment:
Is there a reason you changed the demo time?

Reply:
It was to bump the commit that I had made since it wasn't tracking that I had pulled from `minesweeper`, Also n^2 >> 10n